org.americanmonk.TipitakaPaliReader.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org.americanmonk.TipitakaPaliReader
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tipitaka_pali_reader
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  
+modules:
+  - name: tipitaka_pali_reader
+    buildsystem: simple
+    build-commands:
+      - cp -R tpr/bin/ /app/tpr
+      - chmod +x /app/tpr/tipitaka_pali_reader
+      - mkdir /app/bin/
+      - mkdir /app/lib/
+      - ln -s /app/tpr/tipitaka_pali_reader /app/bin/tipitaka_pali_reader
+      - ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so
+      - install -Dm644 tpr/icons/24x24/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/24x24/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/32x32/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/32x32/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/48x48/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/48x48/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/128x128/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/128x128/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/256x256/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/256x256/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/icons/512x512/org.americanmonk.TipitakaPaliReader.png /app/share/icons/hicolor/512x512/apps/org.americanmonk.TipitakaPaliReader.png
+      - install -Dm644 tpr/org.americanmonk.TipitakaPaliReader.desktop /app/share/applications/org.americanmonk.TipitakaPaliReader.desktop
```

Comment:
I don't see the mandatory appstream file being installed here.

Reply:
what is mandatory app stream?

Reply:
The relevant part in the documentation you acknowledged having read:

https://docs.flathub.org/docs/for-app-authors/requirements/#appstream